DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/hostpipes/src/hostpipes.cpp

```diff
@@ -0,0 +1,216 @@
+#include <CL/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+// Host pipe support in $INTELFPGAOCLSDKROOT/include/sycl/ext/intel/prototype
+#include <host_pipes.hpp>
+
+#include <algorithm>
+#include <iomanip>
+#include <iostream>
+#include <numeric>
+#include <vector>
+
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities//include/dpc_common.hpp
+#include "dpc_common.hpp"
```

Comment:
```suggestion
#include "exception_handler.hpp"
```

And please remove the comment above about where to find dpc_common